modules/deployment-plug-in-cluster.adoc

```diff
@@ -10,32 +10,79 @@ After pushing an image with your changes to a registry, you can deploy the plug-
 
 .Procedure
 
-. To deploy your plug-in to a cluster, instantiate your plug-in by running the following command:
+. To deploy your plug-in to a cluster, install a Helm chart with the name of the plug-in as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option. Provide the location of the image within the `plugin.image` parameter by using the following command:
+
 +
 [source,terminal]
 ----
-$ oc process -f template.yaml \
-  -p PLUGIN_NAME=my-plugin \ <1>
-  -p NAMESPACE=my-plugin-namespace \ <2>
-  -p IMAGE=quay.io/my-repository/my-plugin:latest \ <3>
-  | oc create -f -
+helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location
 ----
-<1> Update with the name of your plug-in.
-<2> Update with the namespace.
-<3> Update with the name of the image you created.
-+
-This command runs a light-weight NGINX HTTP server to serve the assets for your plug-in.
 
-[IMPORTANT]
-====
-`PLUGIN_NAME` must match the plug-in name you used in the `consolePlugin` declaration of `package.json`.
-====
+. Optional: You can specify additional parameters by using the values set of supported parameters. 
 
-[start=2]
-. Patch the *Console Operator* configuration to enable the plug-in by running the following command:
 +
-[source,terminal]
-
+[source,yaml]
 ----
-$ oc patch consoles.operator.openshift.io cluster --patch '{ "spec": { "plugins": ["my-plugin"] } }' --type=merge
+plugin:
+  name: ""
+  description: ""
+  image: ""
+  imagePullPolicy: IfNotPresent
+  replicas: 2
+  port: 9443
+  securityContext:
+    enabled: true
+  podSecurityContext:
+    enabled: true
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  containerSecurityContext:
+    enabled: true
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+        - ALL
+  resources:
+    requests:
+      cpu: 10m
+      memory: 50Mi
+  basePath: /
+  certificateSecretName: ""
+  serviceAccount:
+    create: true
+    annotations: {}
+    name: ""
+  patcherServiceAccount:
+    create: true
+    annotations: {}
+    name: ""
+  jobs:
+    patchConsoles:
+      enabled: true
+      image: "registry.redhat.io/openshift4/ose-tools-rhel8@sha256:e44074f21e0cca6464e50cb6ff934747e0bd11162ea01d522433a1a1ae116103"
+      podSecurityContext:
+        enabled: true
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      containerSecurityContext:
+        enabled: true
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL
+      resources:
+        requests:
+          cpu: 10m
+          memory: 50Mi
 ----
+
+.Verification
```

Comment:
The correct nav path is:
Administration -> Cluster Settings -> Configuration -> Console(operator.openshift.io) -> Console plugins
<img width="820" alt="Screenshot 2023-01-09 at 2 01 54 PM" src="https://user-images.githubusercontent.com/12692381/211249495-2adb7693-2c23-4f97-8c71-d5e017b2f11f.png">